debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -85,11 +86,11 @@ public SchemaBuilder schemaBuilder(Column column) {
             return Year.builder();
         }
         if (matches(typeName, "ENUM")) {
-            String commaSeparatedOptions = extractEnumAndSetOptions(column, true);
+            String commaSeparatedOptions = extractEnumAndSetOptions(column);
```

Comment:
@pranmitt, just for reference, this `schemaBuilder(Column)` method is called only when the table schema changes (e.g., because of a DDL statement), and this is the method that ultimately calls the DDL parser method.